Yoast/javascript
# Remove last circular dependency

## Summary

* `search-metadata-previews` still had a dependency on `yoast-components`. After removing some imports from `yoast-components` related to `react-intl` I could drop this dependency.

## Relevant technical choices:

* The `react-intl` files I deleted are no longer used.

## Test instructions

This PR can be tested by following these steps:

* Make sure search metadata preview still work

## UI changes

## Quality assurance
